source/operatingsystems/windows/activedirectory/resetpassword.md

```diff
@@ -0,0 +1,43 @@
+# How to Reset a User Password in Active Directory
+
+This guide will show you how you can reset a user's password with Active Directory, you will need to gain access to the Domain Controller within your network to follow this guide.
+
+I will be providing two examples of resetting a user's password, one using the `Active Directory Users and Computers` tool and the other using `Powershell`.
+
+## Using the Active Directory Users and Computers Tool
+
+- You will need to first log on to the Domain Controller with domain administrator credentials.
+- Launch the `Active Directory Users and Computers` Tool, you can do this by going to `Start Menu > Windows Administrative Tools > Active Directory Users and Computers`.
+
+You will then be presented with a window that looks similar to the below:
+
+![AD Users and Computers](files/resetpassword/AD_userandcomputers.PNG)
+
+- Open the `Users` folder within this window.
+- Right-click on the user that you would like to reset the password for.
+- Click the `Reset Password...` option.
+
+A window similar to the below will then be displayed:
+
+![AD reset](files/resetpassword/AD_reset.PNG)
+
+- Within this window, enter a `New password` for the user.
+- If you would like the user to change the password upon next logging on, check the `User must change password at next logon` check box.
```

Comment:
This may cause problems when they attempt to connect over RDP with NLA enabled (which should be always). I would recommend UKFast client's always clear this checkbox but instruct users to update their password after they've logged in via ctrl+alt+insert etc.

Reply:
fc9c54a8f13df22c578962e250349633f9516b0c